### PR TITLE
Adding meta label to cinder, nova, designate metrics

### PR DESCRIPTION
**Pull request by rajivmucheli**
@auhlig 

I am not sure if adding check will help, since it doesnt have any value. I have shared the screenshot to you, the below may work ? : 

meta = "{{ $labels.service }}"

### CODE REVIEW

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 1]
```diff
@@ -1,27 +1,29 @@
 ### Networking-DVS Alerts ###
 
 ALERT OpenstackDVSPolling
-  IF (min(rate(openstack_networking_dvs_plugins_ml2_drivers_mech_dvs_agent_dvs_agent_process_ports_timer_count[5m])) by (region,kubernetes_pod_name) *60) < 1
+  IF (min(rate(openstack_networking_dvs_plugins_ml2_drivers_mech_dvs_agent_dvs_agent_process_ports_timer_count[5m])) by (region,kubernetes_pod_name,check) *60) < 1
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: remove outer brackets. region is not neccessary.

File: system/kube-monitoring/charts/prometheus-frontend/openstack-designate.alerts [hunk 2]
```diff
@@ -1,44 +1,47 @@
 ### Designate
 
 ALERT OpenstackDesignateDnsBindDown
-  IF max(bind_up) by (region, kubernetes_name) < 1.0
+  IF max(bind_up) by (region, kubernetes_name,check) < 1.0
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-designate.alerts]: region is not neccessary

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
+    meta = "{{ $labels.check }}"
   }
   ANNOTATIONS {
     summary = "Polling iterations fall below 1 per minute",
     description = "Polling iterations of DVS Agent {{ $labels.kubernetes_pod_name }} is below 1 per minute.",
   }
 
 ALERT OpenstackDVSSecGroup
-  IF max(openstack_networking_dvs_security_group_updates_timer{quantile="0.99"}) by (region,kubernetes_pod_name) > 60
+  IF max(openstack_networking_dvs_security_group_updates_timer{quantile="0.99"}) by (region,kubernetes_pod_name,check) > 60
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: region is not neccessary

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 4]
```diff
@@ -31,113 +33,121 @@ ALERT OpenstackDVSSecGroup
 ### Nova Metrics ###
 
 ALERT OpenstackNovaMaxDiskUsagePerc
-  IF 1.0 - min(openstack_compute_nodes_free_disk_gb_gauge{host!~".*ironic.*"}/openstack_compute_nodes_local_gb_gauge) by (region,availability_zone) > .90
+  IF 1.0 - min(openstack_compute_nodes_free_disk_gb_gauge{host!~".*ironic.*"}/openstack_compute_nodes_local_gb_gauge) by (region,availability_zone,check) > .90
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: rm region

File: system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts [hunk 5]
```diff
@@ -31,113 +33,121 @@ ALERT OpenstackDVSSecGroup
...
+    meta = "{{ $labels.check }}"
   }
   ANNOTATIONS {
     summary = "Nova Maximum Disk Usage percentage metric",
     description = "Nova Maximum Disk Usage is above 90%",
   }
 
 ALERT OpenstackNovaMaxRAMUsagePerc
-  IF 1.0 - min(openstack_compute_nodes_free_ram_mb_gauge{host!~".*ironic.*"}/openstack_compute_nodes_memory_mb_gauge) by (region,availability_zone) > .95
+  IF 1.0 - min(openstack_compute_nodes_free_ram_mb_gauge{host!~".*ironic.*"}/openstack_compute_nodes_memory_mb_gauge) by (region,availability_zone,check) > .95
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-nova.alerts]: rm region

File: system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts [hunk 6]
```diff
@@ -1,13 +1,14 @@
 ### Cinder Alerts ###
 
 ALERT OpenstackCinderVolumeStuck
-  IF (sum(openstack_stuck_volumes_count_gauge) by (host,status)) > 0
+  IF sum(openstack_stuck_volumes_count_gauge) by (host,status,check) > 0
```
Comment (Carthaca) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts]: afaik there is no `check` label

File: system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts [hunk 7]
```diff
@@ -1,13 +1,14 @@
...
   FOR 5m
   LABELS {
     service = "cinder",
     severity = "info",
     tier = "openstack",
-    dashboard = "cinder"
+    dashboard = "cinder",
+    meta = "{{ $labels.check }}"
```
Comment (Carthaca) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-cinder.alerts]: I suggest `meta = "{{ $value }} volumes"` to show the number of affected stuck volumes

**Comment by rajivmucheli** [comment]
Closing PR due to many conflicts

**Pull request closed without merging**